engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -102,9 +102,11 @@ class WorkflowManagerActor(config: Config)
 
   private val donePromise = Promise[Unit]()
 
+  private val conf = ConfigFactory.load
+
   override def preStart() {
     addShutdownHook()
-    restartIncompleteWorkflows()
+    if (conf.getBoolean("system.workflow-restart")) { restartIncompleteWorkflows() }
```

Comment:
I'll just bring it up after standup and we can hash it out. And yes @scottfrazer I'll start making the change to 0.19_hotfix.